modules/running-compliance-scans.adoc

```diff
@@ -0,0 +1,113 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/compliance-scans.adoc
+
+[id="running-compliance-scans_{context}"]
+= Running compliance scans
+
+Running a scan using the CIS profiles. For convenience, the Compliance Operator creates a `ScanSetting` object with reasonable defaults on startup. This `ScanSetting` object is named `default`.
+
+.Procedure
+
+.  Inspect the `ScanSetting` object by running:
++
+[source,terminal]
+----
+$ oc describe scansettings default
+----
```

Comment:
It would be good if we add `-n openshift-compliance` in all command mentioned in this section.